x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/Subject.java

```diff
@@ -87,7 +87,7 @@ public Map<String, Object> getMetadata() {
         return metadata;
     }
 
-    public Version getVersion() {
+    public TransportVersion getTransportVersion() {
```

Comment:
In other places, when replacing existing version methods, I've just left it as `getVersion`

Reply:
I tended to replace the method name too. I feel it might read better